pkg/lib/cockpit-po-plugin.js

```diff
@@ -64,8 +65,8 @@ module.exports = class {
         return expr;
     }
 
-    build_patterns(compilation, extras) {
-        const patterns = [
+    build_paths(compilation, extras) {
+        const paths = [
             // all translations for that page, including manifest.json and *.html
             `pkg/${this.subdir}.*`,
```

Comment:
Ugh.  This is why we were using a regexp, actually.  This actually is *a pattern*.  :(